test/functional/p2p_blockfilters.py

```diff
@@ -64,11 +63,11 @@ def run_test(self):
         disconnect_nodes(self.nodes[0], 1)
 
         self.nodes[0].generate(1)
-        wait_until(lambda: self.nodes[0].getblockcount() == 1000)
+        node0.wait_until(lambda: self.nodes[0].getblockcount() == 1000)
         stale_block_hash = self.nodes[0].getblockhash(1000)
 
         self.nodes[1].generate(1001)
-        wait_until(lambda: self.nodes[1].getblockcount() == 2000)
+        node1.wait_until(lambda: self.nodes[1].getblockcount() == 2000)
```

Comment:
`node1` is a `mininode`. `self.nodes[1]` is a `TestNode`.

It might be best to add a `def wait_until(self, ...):` to the test framework.

```suggestion
        self.wait_until(lambda: self.nodes[1].getblockcount() == 2000)
```